### ISSUE TITLE
telegram.sendCopy throws exception when message is photo without caption

### ISSUE THREAD

**Issue by ofstudio** [telegraf/telegraf#938]
## Context

* Telegraf.js Version: 3.36.0
* Node.js Version: 12
* Operating System: Alpine Linux

## Expected Behavior

Copy of a message to be sent to another chat.

## Current Behavior

Message is not being sent and an error produced in logs

## Failure Information (for bugs)

`telegram.sendCopy` throws exception when message is photo without `caption` field. If `caption` field exists everything is ok.

Possible related with #896 and #890 

### Steps to Reproduce

Please provide detailed steps for reproducing the issue.

```js
ctx.telegram.sendCopy(chatId, {
    "message_id": 123,
    "from": {
      "id": 123456,
      "is_bot": false,
      "first_name": "xxx",
      "last_name": "xxx",
      "username": "xxx",
      "language_code": "en"
    },
    "chat": {
      "id": 123456,
      "first_name": "xxx",
      "last_name": "xxx",
      "username": "xxx",
      "type": "private"
    },
    "date": 1584033705,
    "photo": [
      {
        "file_id": "xxx-xxx",
        "file_unique_id": "xxx",
        "file_size": 17018,
        "width": 201,
        "height": 320
      },
      {
        "file_id": "xxx-xxx",
        "file_unique_id": "xxx",
        "file_size": 49439,
        "width": 420,
        "height": 669
      }
    ]
})
```

### Failure Logs

```
 message=undefined is not iterable (cannot read property Symbol(Symbol.iterator)), stack=TypeError: undefined is not iterable (cannot read property Symbol(Symbol.iterator))
wcsfan_bot |     at formatHTML (/usr/local/src/bot/node_modules/telegraf/markup.js:167:19)
wcsfan_bot |     at Object.photo (/usr/local/src/bot/node_modules/telegraf/core/replicators.js:110:16)
wcsfan_bot |     at Telegram.sendCopy (/usr/local/src/bot/node_modules/telegraf/telegram.js:410:27)
wcsfan_bot |     at /usr/local/src/bot/controllers/anon.js:157:27
wcsfan_bot |     at processTicksAndRejections (internal/process/task_queues.js:97:5)
wcsfan_bot |     at async Promise.all (index 0)
```

**Comment by ofstudio** [comment]
## Temporary workaround

```js
// Temporary workaround on https://github.com/telegraf/telegraf/issues/938
function _w938(m) {
  if (m.animation || m.audio || m.document || m.photo || m.video || m.voice)
    m.caption = m.caption || ''
  return m
}

// ...  
// ...
ctx.telegram.sendCopy(chatId, _w938(message))

```

**Comment by patrickteivas** [comment]
I can't be sure, but AFAIK fix for #890 should fix this error too, but a version of the NPM package since this fix is not changed tho, so probably it's not pushed to prod (aka NPM registry).

**Comment by wojpawlik** [comment]
Duplicate of #891
